File: src/contest/others/coderunner/2014/sample/python/main.py
import urllib2
import time

def query(url):
	res = urllib2.urlopen(url)
	return res.read()

token = "xxxxxxx"

for i in range(0, 2):
	url    = "http://example.com/api01?token=%s" % token
	result = query(url)
	print result

	time.sleep(1)

File: src/contest/others/coderunner/2014/sample/ruby/main.rb
require 'open-uri'

def query(url)
  res = open(url)
  res.string
end

token  = "xxxxxxx"

2.times.each do |i|
  url    = "http://example.com/api01?token=#{token}"
  result = query(url)
  puts result

  sleep 1
end